diff --git a/testcases/network/nfs/nfs_stress/nfs_lib.sh b/testcases/network/nfs/nfs_stress/nfs_lib.sh
index 0abf39d..dfdec8b 100644
--- a/testcases/network/nfs/nfs_stress/nfs_lib.sh
+++ b/testcases/network/nfs/nfs_stress/nfs_lib.sh
@@ -87,7 +87,7 @@ nfs_setup()
 				*:$remote_dir"
 		fi
 
-		opts="-o proto=$type,vers=$i"
+		opts="-o intr,rsize=1024,wsize=1024,proto=$type,vers=$i"
 
 		if [ $TST_IPV6 ]; then
 			mount_dir="[$(tst_ipaddr rhost)]:$remote_dir"
