diff --git a/testcases/kernel/security/smack/smack_set_onlycap.sh b/testcases/kernel/security/smack/smack_set_onlycap.sh
index 5753d18..69e6e4d 100755
--- a/testcases/kernel/security/smack/smack_set_onlycap.sh
+++ b/testcases/kernel/security/smack/smack_set_onlycap.sh
@@ -23,7 +23,7 @@ start_label=$(cat "$smackfsdir/onlycap" 2>/dev/null)
 echo "$my_label" 2>/dev/null > "$smackfsdir/onlycap"
 
 label=$(cat "$smackfsdir/onlycap" 2>/dev/null)
-if [ "$label" != "$my_label" ]; then
+if [ "$label" != "$my_label " ]; then
 	tst_brkm TFAIL "The smack label reported for \"$smackfsdir/onlycap\" "
 		       "is \"$label\", not the expected \"$my_label\"."
 fi
