diff --git a/testcases/kernel/security/integrity/ima/tests/ima_violations.sh b/testcases/kernel/security/integrity/ima/tests/ima_violations.sh
index 1b86b5f..e3b223b 100755
--- a/testcases/kernel/security/integrity/ima/tests/ima_violations.sh
+++ b/testcases/kernel/security/integrity/ima/tests/ima_violations.sh
@@ -62,7 +62,7 @@ close_file_write()
 
 init()
 {
-	service auditd status > /dev/null 2>&1
+	systemctl status auditd > /dev/null 2>&1
 	if [ $? -ne 0 ]; then
 		log=/var/log/messages
 	else
