diff --git a/testcases/network/stress/ssh/ssh-stress03-rmt b/testcases/network/stress/ssh/ssh-stress03-rmt
index 44077ec..da1abeb 100644
--- a/testcases/network/stress/ssh/ssh-stress03-rmt
+++ b/testcases/network/stress/ssh/ssh-stress03-rmt
@@ -51,7 +51,7 @@ case $ip_ver in
 ;;
 6)
 	localhost="::1"
-	ssh -6 -f -N -L $lport/[$server_ipaddr]/$rport \
+	ssh -6 -f -N -L $lport:[$server_ipaddr]:$rport \
 		root@$server_ipaddr -F $ssh_config
 ;;
 esac